src/test/java/core/basesyntax/reporter/CsvReportGeneratorTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.reporter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CsvReportGeneratorTest {
+    private static FruitStorageDao fruitStorageDao;
+    private static ReportGenerator csvReportGenerator;
+
+    @BeforeAll
+    static void beforeAll() {
+        fruitStorageDao = new FruitStorageDaoImpl();
+        csvReportGenerator = new CsvReportGenerator(fruitStorageDao);
+    }
+
+    @Test
+    void generateReport_withEmptyData_ok() {
+        String expectedReport = "fruit,quantity";
+        assertEquals(expectedReport, csvReportGenerator.generateReport());
+    }
+
+    @Test
+    void generateReport_withNonEmptyData_notOk() {
```

Comment:
Test name says something should not be ok, but everyting seems to be ok with this flow